docs/pt_BR/README.md

```diff
@@ -0,0 +1,82 @@
+* Habilidade para produzir saidas para browsers
+* Acessivel [NPM scripts](/npm_scripts.md)
+* Usa o [webpack](https://github.com/webpack/webpack) e [vue-loader](https://github.com/vuejs/vue-loader) com Hot Module Replacement
+* Automatização do reinicio dos processos `main` do electron
+* HTML/CSS/JS pre-processor suporte com [vue-loader](https://github.com/vuejs/vue-loader/)
+* ES6 com [`stage-0`](https://babeljs.io/docs/plugins/preset-stage-0/) by default
+* Uso de [`babili`](https://github.com/babel/babili) para remover a nescessidade de fazer transpiling completo para o ES5
+* ESLint \(com suporte ao [`standard`](https://github.com/feross/standard) e [`airbnb-base`](https://github.com/airbnb/javascript)\)\*
+* Teste unitario com \(with [`karma`](https://github.com/karma-runner/karma) + [`mocha`](https://github.com/mochajs/mocha)\)\*
+* End-to-end Testing \(with [`spectron`](https://github.com/electron/spectron) + [`mocha`](https://github.com/mochajs/mocha)\)\*
+
+\*Customizavel durante o uso da `vue-cli`
+
+### Introdução
+
+Esse template foi construído utilizando [`vue-cli`](https://github.com/vuejs/vue-cli) e inclui opções para customizar a estrutura final do seu app. O template utiliza `node@^7` ou superior como mínimo. electron-vue também recomenda utilizar o [`yarn`](https://yarnpkg.org), gerenciador de pacotes que manipula seus pacotes de uma forma muito melhor e vai ajudar a reduzir o peso final do projeto `yarn clean`.
+
+```bash
+# Instala o vue-cli e o template do projeto
+npm install -g vue-cli
+vue init simulatedgreg/electron-vue my-project
+
+# Instala as dependencias e executa seu app (your app)
+cd my-project
+yarn # ou npm install
+yarn run dev # ou npm run dev
+```
+
+##### Você e um usuario de Windows
+
+Por favor olhe a [**Uma Nota Para Usuarios de Windows**](https://simulatedgreg.gitbooks.io/electron-vue/content/en/getting_started.html#a-note-for-windows-users) para que você tenha certeza que esta com todas as dependencias instaladas para conseguir fazer deploy e construir a aplicação.
+
+##### Querendo usar Vue 1?
+
+So um ponto a ser dado, a suporte ao Vue 1 foi oficialmente descontinuado então a estrutura do projeto, features e documentação vão refletir essas mudanças \([**documentação legado**](https://github.com/SimulatedGREG/electron-vue/tree/1.0/docs)\).
+
+```bash
+vue init simulatedgreg/electron-vue#1.0 my-project
+```
+
+### Proximos Passos
+
+Certifique-se de olhar a [documentação](https://simulatedgreg.gitbooks.io/electron-vue/content/). Aqui você vai achar informação util sobre a configuração, estrutura do projeto, e construção de sua aplicação. Tambe tem a acessivel [FAQs](https://simulatedgreg.gitbooks.io/electron-vue/content/en/faqs.html), que ajuda a responder questões ja respondidas.
+
+## Criado usando electron-vue
+
+De uma olhada nesses projetos impressionantes, usando electron-vue. Que que seu projeto seja listado ? Se sinta livre de enviar um pull request.
```

Comment:
- Change `De uma olhada nesses projetos impressionantes, usando electron-vue. Que que seu projeto seja listado ? Se sinta livre de enviar um pull request.` to `De uma olhada nesses projetos impressionantes usando electron-vue. Quer que seu projeto seja listado aqui? Sinta-se livre de enviar um pull request.`